src/app/frontend/deploy/helpsection/userhelp.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+@import "../../../assets/styles/color_variables";
+
+.kd-user-help * {
+  color: gray;
```

Comment:
Actually not only colors, but font sizes are also good candidates for extraction. Colors like black/gray/white are used in many places right now. It would be great to have some generic names for them that could be applied to elements, otherwise multiple css files will end up with same color variables just with different names.